### PR TITLE
Update readme.txt for v1.5.0

### PR DESCRIPTION
**Pull request by dreamofabear**

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -108,6 +108,15 @@ Web Stories are powered by [AMP](https://amp.dev/), which adds some restrictions
 
 For the plugin's full changelog, please see [the Releases page on GitHub](https://github.com/google/web-stories-wp/releases).
 
+= 1.5.0 =
+
+**Release Date:** March 18, 2021.
+
+* New feature: new Gutenberg blocks for displaying your stories.
```
Comment (swissspidy) [inline readme.txt]:
```suggestion
* New feature: new enhanced Gutenberg block for displaying your stories.
```

It's a single block. Might also want to mention the other new features of this project. So more like

```suggestion
* New feature: new enhanced Gutenberg block, widget and theme integrations for displaying your stories in creative ways.
```

Comment (swissspidy) [reply readme.txt]: @amedina perhaps you have some good suggestion for the changelog item here?

**Comment by dreamofabear** [comment]
How about:

> New feature: new ways to showcase your stories with an enhanced Gutenberg block, widget, and theme integrations.

**Comment by swissspidy** [comment]
> How about:
> 
> > New feature: new ways to showcase your stories with an enhanced Gutenberg block, widget, and theme integrations.

sgtm!

**Review by o-fernandez — APPROVED** [review]

**Comment by dreamofabear** [comment]
Ignoring flaky E2E instead of re-running and wasting CI resources.

**Pull request merged**